CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ All notable changes to the Zlux App Manager will be documented in this file.
 
 ## `2.18.3`
 
-- Enhancement: Upgraded dependency `sass` to version `1.93.2` to avoid some security risks.
+- Enhancement: Upgraded dependency `sass` to version `1.93.2` to avoid some security risks. ([#677](https://github.com/zowe/zlux-app-manager/pull/677))
+- Enhancement: Exclude development-only npm packages from production builds. ([#678](https://github.com/zowe/zlux-app-manager/pull/678))
```

Comment:
I am not sure, if this will make it into 2.18.3, the RC3 was already made and looking good so far. If not, just update the version to 2.18.4.